api/src/main/java/org/pmiops/workbench/billing/BillingGarbageCollectionService.java

```diff
@@ -137,13 +137,12 @@ private void transferOwnership(final String projectName) {
     fireCloudService.addOwnerToBillingProject(garbageCollectionSA, projectName);
 
     try {
-      final GoogleCredential gcsaCredential =
+      final GoogleCredentials gcsaCredential =
```

Comment:
Can we factor it out a bit more and inject a `Provider<GoogleCredentials>` that always gives you refreshed ones? Having to call `refresh()` like this is kinda temporal coupling, and in any case, we shouldn't expect a programmer to remember to do something we're not willing to ask the computer to remember. So if the provider doesn't work, we can at least have a helper method that fetches and refreshes in one shot and returns one ready to go, RAII-style.